mgmtsystem_nonconformity_project/README.rst

```diff
@@ -0,0 +1,10 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===========================================
+Management System - Nonconformity - Project
+===========================================
+
+This module enables you to set a project as an action in the nonconformity.
+
```

Comment:
Can you transform the readme to the new OCA format: splitted in chunks in a `readme` folder?

https://github.com/OCA/maintainer-tools/tree/master/template/module/readme